core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -1740,28 +2480,167 @@ public void testGetAllStoreSomeMatchCacheAccessExceptionBeforeLoaderAllMatchNone
    * </ul>
    */
   @Test
-  public void testGetAllStoreSomeMatchCacheAccessExceptionAfterLoaderAllMatchNoneFail() throws Exception {
-    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B), Collections.singleton("keyA3"));
+  public void testGetAllStoreSomeMatchCacheAccessExceptionAfterLoaderAllMatchNoneFail() throws Exception {
+    final FakeStore fakeStore = new FakeStore(getEntryMap(KEY_SET_A, KEY_SET_B), Collections.singleton("keyA3"));
+    this.store = spy(fakeStore);
+
+    final FakeCacheLoaderWriter fakeLoader = new FakeCacheLoaderWriter(getEntryMap(KEY_SET_A, KEY_SET_C, KEY_SET_D));
```

Comment:
testGetAllStoreSomeMatchCacheAccessExceptionAfterLoaderAllMatchNoneFail() method is not changed at all. GIT diff is reporting it incorrectly.
However, such issues needs to be fixed and I propose opening a new issue for it.